coralproject/talk
# Keyboard shortcuts

## What does this PR do?

Re-enables keyboard shortcuts in the admin panel.

## How do I test this PR?

1. Go to the admin panel.
2. Press `?`, you should see a keyboard shortcut popup.
3. All keyboard shortcuts should work.
